src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.Initialization.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
```

Comment:
Unnecessary using copied and pasted. Could cause problems when included in other projects. Will fix.